proto/buf.gen.yaml

```diff
@@ -5,7 +5,7 @@ version: v1
 managed:
   enabled: true
   go_package_prefix:
-    default: go.buf.build/conduitio/conduit/conduitio/conduit
+    default: "github.com/conduitio/conduit/proto/gen"
     except:
       - buf.build/googleapis/googleapis
     override:
```

Comment:
One last thing - I propose we change the line below to this:
```
      buf.build/grpc-ecosystem/grpc-gateway: go.buf.build/protocolbuffers/go/grpc-ecosystem/grpc-gateway
```
In other words, we should use the [`protocolbuffers/go`](https://buf.build/protocolbuffers/templates/go) template instead of `conduitio/conduit`, the buf team keeps that template updated in case new versions of the protoc-gen-go plugin are released ([doc](https://docs.buf.build/bsr/remote-generation/remote-plugin-execution#protoc-based-plugins)).

We need to regenerate the proto code and run `go mod tidy` after this.